eclass/xdg-utils.eclass

```diff
@@ -7,7 +7,7 @@
 # freedesktop-bugs@gentoo.org
 # @AUTHOR:
 # Original author: Gilles Dartiguelongue <eva@gentoo.org>
-# @SUPPORTED_EAPIS: 0 1 2 3 4 5 6 7
+# @SUPPORTED_EAPIS: 0 1 2 3 4 5 6 7 8
```

Comment:
Drop EAPIs 0 to 4? Keeping them doesn't make much sense, especially since `xdg.eclass` no longer supports 0 to 3 either.
```suggestion
# @SUPPORTED_EAPIS: 5 6 7 8
```